components/Navigation/Navigation.js

```diff
@@ -58,21 +36,26 @@ export const Navigation = () => {
           />
         </button>
       </div>
-      <nav className="flex-1 hidden lg:flex" aria-label="Hlavní menu">
-        <ul className="flex-1 flex justify-end items-center divide-x-2 divide-accent-500">
-          <li>
+      <nav
+        className={`${hiddenMobileMenu}
+      bg-accent-200 absolute top-[52px] md:top-[76px] right-0 z-50
+      lg:bg-transparent lg:relative lg:flex-1 lg:flex lg:top-0`}
+        aria-label="Hlavní menu"
+      >
```

Comment:
Or you could move all of it here 
```suggestion
      <nav
        className={classNames({ hidden: !isMenuOpen }, 'bg-accent-200 absolute top-[52px] md:top-[76px] right-0 z-50 lg:bg-transparent lg:relative lg:flex-1 lg:flex lg:top-0')}
        aria-label="Hlavní menu"
      >
```

or leave it as is. No strong preference